eo.json

```diff
@@ -317,14 +317,14 @@
         "Account.Register": "Registriĝi",
         "Account.LoginOrRegister": "Ensaluti / Registriĝi",
         "Account.LogoutConfirm": "Ĉu vere elsaluti?",
-        "Account.LoggingIn": "Ensaluti ...",
-        "Account.LoggingOut": "Elsaluti ...",
-        "Account.Registering": "Registrante ...",
+        "Account.LoggingIn": "Ensaluti...",
+        "Account.LoggingOut": "Elsaluti...",
+        "Account.Registering": "Registrante...",
         "Account.Anonymous": "Anonima",
         "Account.PatreonSupporter": "Patreon-Subtenanto",
         "Account.Username": "Uzantnomo:",
         "Account.Email": "Retpoŝto:",
-        "Account.RememberMe": "Memoru Min (dum 7 tagoj):",
+        "Account.RememberMe": "Memori Min (dum 7 tagoj):",
```

Comment:
Mi ne certas, kiu modo estus pli bona. Laŭ Komputeko, la -i modo estas uzata, kaj tio estas kion mi anticipis post mi legis la [dokumentadon por Mozilla tradukistoj](https://mozilla-l10n.github.io/styleguides/eo/). Tamen, kaj Lernu](https://lernu.net/eo/ensaluti) kaj [Edukado](https://edukado.net/ext/ensaluti?pagho=ensaluti&reen=hejmo) uzas la -u modon, kaj je mia miro la [Universala Esperanto-Asocio](https://uea.org/) uzas la -u modon ĉie en ĝia retpaĝaro. Se oni konsideras la historion de uzantinterfacojn kiel la teksta uzantinterfaco, kiun oni laŭlitere ordonas la komputilon, ke ĝi faru ion, oni anticipus, ke la -u modo estus uzita, sed la -i modo ŝajnas pli kutime en la programaro. Kion vi pensas?

I'm unsure which form would be best. Komputeko lists the -i form, and this is what I would expect from what I read in the [Mozilla style guide](https://mozilla-l10n.github.io/styleguides/eo/). However, both [Lernu](https://lernu.net/eo/ensaluti) and [Edukado](https://edukado.net/ext/ensaluti?pagho=ensaluti&reen=hejmo) use the -u form, and as a complete surprise, the [Universal Esperanto Association](https://uea.org/) uses the -u form everywhere on their site. Considering the history of user interfaces such as the command line in which you literally command the computer to do something, I would expect the -u form, but the -i form appears to be more common in software. What are your thoughts?

Reply:
There is no problem if you choose to use one or the other. As you saw, different organizations decided to use one or the other, and at the bottom, that decision was taken by people like you that met the same question.

Still, I don't think there will be really a strong need to stadarize in such a detailed level. So don't worry too much about this, and choose the one you prefer. The most important thing is that the user understands the message.

As I said, I would prefer this message with -u, because I feel that it would be clearer to the user that what they actually do with that button is ask the system to remember them. With -i, it sounds to me like what you would want to do is ask yourself to remember yourself. Doesn't sounds good to me, but it could make sense to someone else.

Anyhow, it's ok to me if you want to change it or not. I agree with your decision, whichever it is

Reply:
Alright, I'll go with -u for this one.